client/verta/verta/client.py

```diff
@@ -158,22 +159,23 @@ def __init__(self, host=None, port=None, email=None, dev_key=None,
 
         # verify connection
         conn = _utils.Connection(scheme, socket, auth, max_retries, ignore_conn_err)
-        try:
-            response = _utils.make_request("GET",
-                                           "{}://{}/api/v1/modeldb/project/verifyConnection".format(conn.scheme, conn.socket),
-                                           conn)
-        except requests.ConnectionError:
-            six.raise_from(requests.ConnectionError("connection failed; please check `host` and `port`"),
-                           None)
-
-        def is_unauthorized(response): return response.status_code == 401
-
-        if is_unauthorized(response):
-            auth_error_msg = "authentication failed; please check `VERTA_EMAIL` and `VERTA_DEV_KEY`"
-            six.raise_from(requests.HTTPError(auth_error_msg), None)
-
-        _utils.raise_for_http_error(response)
-        print("connection successfully established")
+        if (connect):
+          try:
+              response = _utils.make_request("GET",
+                                             "{}://{}/api/v1/modeldb/project/verifyConnection".format(conn.scheme, conn.socket),
+                                             conn)
+          except requests.ConnectionError:
+              six.raise_from(requests.ConnectionError("connection failed; please check `host` and `port`"),
+                             None)
+
+          def is_unauthorized(response): return response.status_code == 401
+
+          if is_unauthorized(response):
+              auth_error_msg = "authentication failed; please check `VERTA_EMAIL` and `VERTA_DEV_KEY`"
+              six.raise_from(requests.HTTPError(auth_error_msg), None)
+
+          _utils.raise_for_http_error(response)
+          print("connection successfully established")
```

Comment:
This should indented by four spaces instead of two spaces
```suggestion
            try:
                response = _utils.make_request("GET",
                                               "{}://{}/api/v1/modeldb/project/verifyConnection".format(conn.scheme, conn.socket),
                                               conn)
            except requests.ConnectionError:
                six.raise_from(requests.ConnectionError("connection failed; please check `host` and `port`"),
                               None)

            def is_unauthorized(response): return response.status_code == 401

            if is_unauthorized(response):
                auth_error_msg = "authentication failed; please check `VERTA_EMAIL` and `VERTA_DEV_KEY`"
                six.raise_from(requests.HTTPError(auth_error_msg), None)

            _utils.raise_for_http_error(response)
            print("connection successfully established")
```